tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -1126,14 +1127,43 @@ public function theUserHasCreateAFolderUsingTheGraphApi(
 		string $folder,
 		string $spaceName
 	): void {
-		$this->theUserCreatesAFolderUsingTheGraphApi($user, $folder, $spaceName);
-
+		$folder = \trim($folder, "/");
+		$exploded = explode('/', $folder);
+		$path = '';
+		for ($i = 0; $i < \count($exploded); $i++) {
+			$path = $path . $exploded[$i] . '/';
+			$response = $this->theUserCreateAFolderToAnotherOwnerSpaceUsingTheGraphApi($user, $path, $spaceName);
+		}
 		$this->featureContext->theHTTPStatusCodeShouldBe(
 			201,
-			"Expected response status code should be 201"
+			"Expected response status code should be 201",
+			$response
 		);
 	}
 
+	/**
+	 * @param string $user
+	 * @param string $folder
+	 * @param string $spaceName
+	 * @param string $ownerUser
+	 *
+	 * @return ResponseInterface
+	 *
+	 * @throws GuzzleException
+	 */
+	public function theUserCreateAFolderToAnotherOwnerSpaceUsingTheGraphApi(
```

Comment:
does it reflect the action?
```suggestion
	public function createFolderInSpace(
```